main.js

```diff
@@ -30,5 +30,29 @@ client.on('ready', () => {
         }
     });
 });
+client.on('message', message => { //Ten event wykonuje się, gdy bot wykryje wiadomość
+    const pingEmbed = new Discord.MessageEmbed()
+	.setColor('#eb1540')
+	.setTitle('Nie wołaj wszystkich!')
+    .setThumbnail('https://cdn.discordapp.com/attachments/617673807213232128/852887484542615572/Pingsock.png')
+    .setDescription('\nRozumiemy, że potrzebujesz pomocy, ale nie wszyscy chcą zostać o tym powiadomieni.\n Jest nas tutaj dużo i nie ma sensu, aby każdy dostał bezpośrednio taką informację.\n Nie trudno sobie wyobrazić jak irytujące byłoby, gdyby każdy wołał wszystkich do każdego tematu.\n Dlatego zadaj pytanie i po prostu poczekaj - jeśli ktoś będzie wiedział i mógł, to na pewno spróbuje odpowiedzieć.');
 
+    if (message.content.includes("@everyone")) { //Jeśli wiadomość zawiera @everyone..
+        try{
+            if(!message.member.hasPermission('MENTION_EVERYONE')) //Sprawdzamy, czy wysyłający ma uprawnienia do wzmianki @everyone, jeśli nie to bot wysyła wiadomość
+                message.author.send(pingEmbed)
+        }catch{
+            console.log(message.author.username + " wysłał ping w wiadomości prywatnej do bota, hmm..")
```

Comment:
te console logi chyba też zbędne? Jak ktoś próbował wysłać do bota, to bym pomijał to i nie informował, nieistotna informacja